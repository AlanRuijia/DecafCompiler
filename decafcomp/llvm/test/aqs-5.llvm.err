; ModuleID = 'QuickSort'

define void @initList(i32) {
entry:
  %size = alloca i32
  store i32 %0, i32* %size
  %i = alloca i32
  store i32 0, i32* %i
  ret void
}
