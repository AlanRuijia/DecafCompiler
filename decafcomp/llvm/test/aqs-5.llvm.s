	.section	__TEXT,__text,regular,pure_instructions
	.macosx_version_min 10, 12
	.globl	_initList
	.align	4, 0x90
_initList:                              ## @initList
	.cfi_startproc
## BB#0:                                ## %entry
	movl	%edi, -4(%rsp)
	movl	$0, -8(%rsp)
	retq
	.cfi_endproc


.subsections_via_symbols
