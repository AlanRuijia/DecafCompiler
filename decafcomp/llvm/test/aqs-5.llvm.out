Inserting method decl into symbol table...
Inserting block into symbol table...
Generating program...
Generating package...

Symbol Table: i size 
Symbol Table: initList 
Symbol Table: 
Debug message: Generating MethodDecl...
In IDTypeStringSpecialAST
Allocating size
Serching table...
FetchedObject: Found size
Serching table...
FetchedObject: Found size
Method Block CodeGen...
Block CodeGen......
Serching table...
FetchedObject: Found i
Allocating i
Finished Block CodeGen......
Finish Method Block CodeGen...
Generating Dummy Return void...Finished...
Finish Method CodeGen...

Symbol Table: initList 
Symbol Table: 
